acrobat/blocks/unity/unity.js

```diff
@@ -0,0 +1,23 @@
+import { loadStyle } from '../../scripts/utils.js';
+
+function getUnityLibs(prodLibs = '/unitylibs') {
+  const { hostname } = window.location;
+  if (!hostname.includes('hlx.page')
+    && !hostname.includes('hlx.live')
+    && !hostname.includes('localhost')) {
+    return prodLibs;
+  }
+  const branch = new URLSearchParams(window.location.search).get('unitylibs') || 'main';
+  if (branch.indexOf('--') > -1) return `https://${branch}.hlx.live/unitylibs`;
+  return `https://${branch}--unity--adobecom.hlx.live/unitylibs`;
+}
+
+export default async function init(el) {
+  const unitylibs = getUnityLibs();
+  const stylePromise = new Promise((resolve) => {
+    loadStyle(`${unitylibs}/core/styles/styles.css`, resolve);
+  });
+  await stylePromise;
+  const { default: wfinit } = await import(`${unitylibs}/core/workflow/workflow.js`);
+  await wfinit(el, 'acrobat', unitylibs);
```

Comment:
@saragajic11 please pass 'v2' as the last parameter of this function as acrobat will be using the new refactored code for loading unity.
We added the version parameter as unity block for photoshop is live using the old version for now.

cc @aishwaryamathuria @sanjayms01

Reply:
@Ruchika4 done: https://github.com/adobecom/dc/pull/784/commits/4407a6c42469554c7d1429ea4a7580ff84ac6de2 
thanks!